tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+   "compilerOptions": {
+     "target": "es5",
+     "lib": [
+       "dom",
+       "dom.iterable",
+       "esnext"
+     ],
+     "allowJs": true,
+     "skipLibCheck": true,
+     "esModuleInterop": true,
+     "allowSyntheticDefaultImports": true,
+     "strict": true,
+     "forceConsistentCasingInFileNames": true,
+     "module": "esnext",
+     "moduleResolution": "node",
+     "resolveJsonModule": true,
+     "isolatedModules": true,
+     "downlevelIteration": true,
+     "noEmit": true,
+     "jsx": "preserve"
+   },
+   "include": [
+     "src"
+   ]
+ }
```

Comment:
There is something wrong with `tsconfig.json`, I am not expert enough to figure out what exactly. But here is the config I use. I tested it on clone of your fork, tests are passing after updating tsconfig.

    {
      "compilerOptions": {
        "outDir": "build",
        "module": "esnext",
        "target": "es5",
        "lib": ["es6", "dom", "es2016", "es2017"],
        "sourceMap": true,
        "allowJs": false,
        "jsx": "react",
        "declaration": true,
        "moduleResolution": "node",
        "forceConsistentCasingInFileNames": true,
        "noImplicitReturns": true,
        "noImplicitThis": true,
        "noImplicitAny": true,
        "strictNullChecks": true,
        "suppressImplicitAnyIndexErrors": true,
        "noUnusedLocals": true,
        "noUnusedParameters": true
      },
      "include": ["src"],
      "exclude": ["node_modules", "build", "dist", "example", "rollup.config.js"]
    }

Maybe you will know what exactly is causing the error and update those parts only. Or just copy paste this config, it works pretty well for me.